template/truffle/.gitignore

```diff
@@ -0,0 +1,2 @@
+/build/
```

Comment:
Suggest move this folder under `template/examples` (such as `template/examples/truffle`) as we probably will have multiple examples living in the template for different integration tests.